hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/logical/FullScanLogicalRel.java

```diff
@@ -16,28 +16,32 @@
 
 package com.hazelcast.jet.sql.impl.opt.logical;
 
+import com.hazelcast.sql.impl.calcite.opt.AbstractScanRel;
+import com.hazelcast.sql.impl.calcite.schema.HazelcastTable;
+import com.hazelcast.sql.impl.schema.Table;
 import org.apache.calcite.plan.RelOptCluster;
 import org.apache.calcite.plan.RelOptTable;
 import org.apache.calcite.plan.RelTraitSet;
 import org.apache.calcite.rel.RelNode;
-import org.apache.calcite.rel.core.TableScan;
 
 import java.util.List;
 
-import static java.util.Collections.emptyList;
-
-public class FullScanLogicalRel extends TableScan implements LogicalRel {
+public class FullScanLogicalRel extends AbstractScanRel implements LogicalRel {
```

Comment:
I don't see much value in extending `AbstractScanRel`.

Reply:
Removed redundant `getTableUnwrapped()`